addons/account/models/chart_template.py

```diff
@@ -361,7 +365,17 @@ def deref(values, model):
                     field.type == 'many2one'
                     or (field.type in ('integer', 'many2one_reference') and not value.isdigit())
                 ):
-                    values[fname] = self.ref(value).id if value not in ('', 'False', 'None') else False
+                    try:
+                        values[fname] = self.ref(value).id if value not in ('', 'False', 'None') else False
+                    except ValueError as e:
+                        print(f"Trying to setup: {field}")
+                        print(f"Trying to setup: {field}")
+                        print(f"Trying to setup: {field}")
+                        print(f"Trying to setup: {field}")
+                        print(f"Trying to setup: {field}")
+                        print(f"Trying to setup: {field}")
+                        print(f"Trying to setup: {field}")
+                        raise e
```

Comment:
to remove or to improve?
Would be better to give a feedback on which model and field that failed. (at which step?)